### ISSUE TITLE
[Attract.py] allow multiple --ref structures

### ISSUE THREAD

**Issue by asaladin** [ptools/ptools-legacy#14]
It may be sometimes useful to compare a predicted ligand position to multiple references. I added this issue to check if it's possible to include multiple rmsd checks in the output file without breaking too much tools and backward compatibility.

**Comment by chantalprevost** [comment]
I agree, I needed that several times (and I made various versions of attract.py accordingly)
